packages/codegen/src/commands.ts

```
@@ -0,0 +1,269 @@
+    for (const service of services) {
+      const configFilePath = path.resolve(
+        apiDirectory,
+        service,
+        ConfigurationManager.filename
+      );
+      try {
+        await fs.stat(configFilePath);
+        results.push(service);
+      } catch (error) {
+        // skip projects without a config file
+      }
+    }
+    return results;
+  };
+}
+
+export class ConfigureCommand {
+  public create = async (options: ConfigFileUpdateOptions) => {
+    const directory = await versionFileDirectoryPath(options);
+    const config = new ConfigurationManager({
+      directory: directory,
+    });
+    if (await config.exists()) {
+      throw new Error(
+        `Config already exists - did nothing (did you mean "update"?)`
+      );
+    }
+
+    await config.write(this.defaultConfig(options));
+  };
+
+  public update = async (options: ConfigFileUpdateOptions) => {
+    const directory = await versionFileDirectoryPath(options);
+    const config = new ConfigurationManager({
+      directory: directory,
+    });
+
+    try {
+      await config.delete();
+      await config.write(this.defaultConfig(options));
+    } catch (error) {
+      throw new Error('Config file does not exist - did nothing: ' + error);
+    }
+  };
+
+  private defaultConfig = (
+    options: ConfigFileUpdateOptions
+  ): ConfigurationOptions => {
+    return {
+      version: options.version,
+      service: options.service,
+      scope: typeof options.scope === 'undefined' ? 'local' : options.scope,
+      autoNameInlinedRequest: true,
+      filter: {
+        serviceName: options.service,
+      },
+    };
+  };
+
+  public delete = async (options: ConfigFileOptions) => {
+    const directory = await versionFileDirectoryPath(options);
+    const config = new ConfigurationManager({
+      directory: directory,
+    });
+
+    try {
+      await config.delete();
+    } catch (error) {
+      throw new Error('Config file does not exist - did nothing');
```

This might hide other errors, e.g. a read only file system. I suggest checking if the file exists explicitly rather than assuming all errors are not founds.